src/orders/components/OrderCaptureDialog/messages.ts

```diff
@@ -0,0 +1,146 @@
+import { defineMessages } from "react-intl";
+
+export const messages = defineMessages({
+  title: {
+    id: "0f6YvV",
+    defaultMessage: "Capture Payment",
```

Comment:
question: are we okay with using `"Payment"`? afaik everywhere else we use "Transaction", "Payment" is reserved for the legacy API.

Reply:
I think we should aim to unify that as from the user perspective the underlying API should make no difference. The action is initialised already from the Transactions section which is the section covering the new API. The dialog itself should be unified in terms of naming.